juego.py

```diff
@@ -298,12 +298,12 @@ def load_sound(name, activity_root):
         sound = pygame.mixer.Sound(path)
         return sound
     except:
-        print 'Warning, unable to load: ',path
+        print('Warning, unable to load: ',path)
 
 # Funcion para guardar puntuaciones altas
 def save_puntuacionalta(score, activity_root):
     file_path = os.path.join(activity_root,'data', 'PuntajeAlto')
-    print file_path
+    print(file_path)
```

Comment:
Any reason for printing out the file path?

Reply:
I have demoted logging instead of removing the print statements entirely. : 597d665